card/src/main/java/com/adyen/checkout/card/CardValidationUtils.java

```diff
@@ -44,17 +38,6 @@
     private static final int MAXIMUM_YEARS_IN_FUTURE = 30;
     private static final int MAXIMUM_EXPIRED_MONTHS = 3;
 
-    /**
-     * Checks if the public key for encryption is valid.
-     * @param publicKey The public key string
-     * @return True if valid, False if not.
-     */
-    public static boolean isPublicKeyValid(@Nullable String publicKey) {
```

Comment:
@caiofaustino I removed this assuming we won't need to validate public keys anymore, is that right?

Reply:
yeah that's right